CMakeLists.txt

```diff
@@ -21,9 +21,14 @@ message(STATUS "p4est ${PROJECT_VERSION} "
 # Skip `libsc` if already registered. This is useful when `p4est` is added as a
 # dependency by another software project which in turn already registered `libsc`.
 if(NOT TARGET SC::SC)
-  include(cmake/GitSubmodule.cmake)
-  git_submodule("${PROJECT_SOURCE_DIR}/sc")
-  add_subdirectory(sc)
+  # Option to use an already installed SC library.
+  if ( P4EST_USE_SYSTEM_SC )
+      find_package( SC REQUIRED PATHS /path/to/system/sc )
```

Comment:
I'm not too familiar with CMake, are we really using `/path/to/system/sc` verbatim?

Reply:
Done